### PR TITLE
display pulsar name policies

### PR DESCRIPTION
**Pull request by zxJin-x**
@Shoothzj

### CODE REVIEW

File: lib/api/pulsar/pulsar_namespace_api.dart [hunk 1]
```diff
@@ -52,6 +52,138 @@ class PulsarNamespaceApi {
     return BacklogQuotaResp.fromJson(destinationStorageResp);
   }
 
+  static Future<PolicyResp> getPolicy(String host, int port, String tenant, String namespace) async {
```
Comment (goflutterjava) [inline lib/api/pulsar/pulsar_namespace_api.dart:55]: better to put behind `updateBacklogQuota`, make `getBacklogQuota` close to `updateBacklogQuota`

File: lib/module/pulsar/pulsar_namespace.dart [hunk 2]
```diff
@@ -42,3 +73,58 @@ class BacklogQuotaResp {
...
+  String? topicType;
+  int? defaultNumPartitions;
+
+  AutoTopicCreation(this.isAllowAutoTopicCreation, this.topicType, this.defaultNumPartitions);
+}
+
+class BundlesData {
+  List<String>? boundaries;
+  int? numBundles;
+}
```
Comment (goflutterjava) [inline lib/module/pulsar/pulsar_namespace.dart:130]: add a new line in file

File: windows/flutter/generated_plugin_registrant.cc [hunk 3]
```diff
@@ -2,6 +2,8 @@
 //  Generated file. Do not edit.
 //
 
+// clang-format off
```
Comment (goflutterjava) [inline windows/flutter/generated_plugin_registrant.cc:5]: no need to commit

File: windows/flutter/generated_plugin_registrant.h [hunk 4]
```diff
@@ -2,6 +2,8 @@
 //  Generated file. Do not edit.
 //
 
+// clang-format off
```
Comment (goflutterjava) [inline windows/flutter/generated_plugin_registrant.h:5]: no need to commit

File: lib/module/pulsar/pulsar_namespace.dart (hunk 2 above)
Comment (zxJin-x) [reply lib/module/pulsar/pulsar_namespace.dart:130]: fixed

**Review by goflutterjava — APPROVED** [review]

**Comment by goflutterjava** [comment]
LGTM

**Pull request merged**